index.html

```diff
@@ -330,7 +330,6 @@
                         text: 'Bugs, Issues, Skipped Libcxx Tests',
                     },
                     min: 0,
-                    max: 650,

```

Comment:
Different repos have different scales of data. I feel like letting the table library to decide the perfect max value instead of hard coding